app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/addEmployeeJobInfrmation/index.tsx

```
@@ -6,47 +6,96 @@ import WorkScheduleForm from '../../../../_components/allFormData/workScheduleFo
+    const jobInformationData = {
+      userId: correctUserId,
+      positionId: formValues.positionId || '',
+      branchId: formValues.branchId || '',
+      departmentId: formValues.departmentId || '',
+      employementTypeId: formValues.employementTypeId || '',
+      workScheduleId: formValues.workScheduleId || '',
+      isPositionActive: true,
+      departmentLeadOrNot: formValues.departmentLeadOrNot || false,
+      employmentContractType: formValues.employmentContractType || 'Permanent',
+      jobAction: 'New', // Always send 'New' as string, not ID
+      effectiveStartDate: formValues.effectiveStartDate ? 
```

ESLint

[error] Delete `·?·`

(prettier/prettier)